src/App.tsx

```
@@ -119,24 +92,45 @@ const App = () => {
+            const newPassword = buildPassword(passwordLength, charset);
+            setNewPasswordResult(newPassword);
+        }
+    }, [
+        generatePassword,
+        passwordLength,
+        ,
+        includeLowercase,
+        includeUppercase,
+        includeNumbers,
+        includeSpecialCharacter
+    ]);
```

Biome (1.9.4)

[error] This array contains an empty slot.

Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)